### PR TITLE
endl manipulator.

### PR DESCRIPTION
**Pull request by GarbyXX**
replaced newline character (escape sequence).

\t	Creates a horizontal tab	
\\	Inserts a backslash character (\)	
\"	Inserts a double quote character

### CODE REVIEW

File: print_cmd.cpp [hunk 1]
```diff
@@ -7,11 +7,15 @@ using namespace std: use names for objects and variables from the standard libra
 */
 
 int main() {
-  cout << "Hello GarbyX \n\n";
+  cout << "Hello GarbyX" << endl;
```
Comment (GarbyX) [inline print_cmd.cpp:10]: The endline manipulator serves same use as newline escape character and is well implemented.

File: print_cmd.cpp [hunk 2]
```diff
@@ -7,11 +7,15 @@ using namespace std: use names for objects and variables from the standard libra
...
   cout << "This is a program written in C++";
   return 0;
 }
 
+/*
```
Comment (GarbyX) [inline print_cmd.cpp:15]: Comment indicates actual input.

**Review by GarbyX — APPROVED** [review]
All changes approved.

**Review by GarbyXX — COMMENTED** [review]
endl manipulator has been used.

**Pull request merged**